tests/test-json/H5PasJSON.json

```diff
@@ -0,0 +1,11 @@
+
+{
+    "include": ["src/**/*.json"],
```

Comment:
Do we need this and the `compiler options` below? Doesnt look like it belongs inside H5P content